keyboards/mechlovin/infinity87/rev1/standard/info.json

```diff
@@ -11,115 +11,1378 @@
+                {"label": "H", "matrix": [3, 6], "x": 6.75, "y": 3.25},
+                {"label": "J", "matrix": [3, 7], "x": 7.75, "y": 3.25},
+                {"label": "K", "matrix": [3, 8], "x": 8.75, "y": 3.25},
+                {"label": "L", "matrix": [3, 9], "x": 9.75, "y": 3.25},
+                {"label": ";", "matrix": [3, 10], "x": 10.75, "y": 3.25},
+                {"label": "'", "matrix": [3, 11], "x": 11.75, "y": 3.25},
+                {"label": "#", "matrix": [3, 12], "x": 12.75, "y": 3.25},
+                {"label": "Enter", "matrix": [3, 13], "x": 13.75, "y": 2.25, "w": 1.25, "h": 2},
+
+                {"label": "Shift", "matrix": [4, 0], "x": 0, "y": 4.25, "w": 1.25},
+                {"label": "\\", "matrix": [4, 1], "x": 1.25, "y": 4.25},
+                {"label": "Z", "matrix": [4, 2], "x": 2.25, "y": 4.25},
+                {"label": "X", "matrix": [4, 3], "x": 3.25, "y": 4.25},
+                {"label": "C", "matrix": [4, 4], "x": 4.25, "y": 4.25},
+                {"label": "V", "matrix": [4, 5], "x": 5.25, "y": 4.25},
+                {"label": "B", "matrix": [4, 6], "x": 6.25, "y": 4.25},
+                {"label": "N", "matrix": [4, 7], "x": 7.25, "y": 4.25},
+                {"label": "M", "matrix": [4, 8], "x": 8.25, "y": 4.25},
+                {"label": ",", "matrix": [4, 9], "x": 9.25, "y": 4.25},
+                {"label": ".", "matrix": [4, 10], "x": 10.25, "y": 4.25},
+                {"label": "/", "matrix": [4, 11], "x": 11.25, "y": 4.25},
+                {"label": "Shift", "matrix": [4, 13], "x": 12.25, "y": 4.25, "w": 2.75},
+
+                {"label": "\u2191", "matrix": [4, 15], "x": 16.25, "y": 4.25},
+
+                {"label": "Ctrl", "matrix": [5, 0], "x": 0, "y": 5.25, "w": 1.25},
+                {"label": "GUI", "matrix": [5, 1], "x": 1.25, "y": 5.25, "w": 1.25},
+                {"label": "Alt", "matrix": [5, 2], "x": 2.5, "y": 5.25, "w": 1.25},
+                {"label": "Space", "matrix": [5, 6], "x": 3.75, "y": 5.25, "w": 6.25},
+                {"label": "Alt", "matrix": [5, 10], "x": 10, "y": 5.25, "w": 1.25},
+                {"label": "GUI", "matrix": [5, 11], "x": 11.25, "y": 5.25, "w": 1.25},
+                {"label": "Menu", "matrix": [5, 12], "x": 12.5, "y": 5.25, "w": 1.25},
+                {"label": "Ctrl", "matrix": [5, 13], "x": 13.75, "y": 5.25, "w": 1.25},
+
+                {"label": "\u2190", "matrix": [5, 14], "x": 15.25, "y": 5.25},
+                {"label": "\u2193", "matrix": [5, 15], "x": 16.25, "y": 5.25},
+                {"label": "\u2192", "matrix": [5, 16], "x": 17.25, "y": 5.25}
+            ]
+        },
+        "LAYOUT_tkl_iso_split_bs_rshift": {
+            "layout": [
+                {"label": "Esc", "matrix": [0, 0], "x": 0, "y": 0},
+
+                {"label": "F1", "matrix": [0, 1], "x": 2, "y": 0},
+                {"label": "F2", "matrix": [0, 2], "x": 3, "y": 0},## Issues Fixed or Closed by This PR
+
+* 
+
+
```

Comment:
```suggestion
                {"label": "F2", "matrix": [0, 2], "x": 3, "y": 0},
```

Reply:
Thought I hit Undo before saving that...